centrifuge/src/main/java/io/github/centrifugal/centrifuge/Dns.java

```diff
@@ -1,5 +1,7 @@
 package io.github.centrifugal.centrifuge;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
Does this import works everywhere? Or is it somehow related to IDE environment?

Reply:
So java is still has no nullability support, annotation cannot change that - it's not gonna cause crashes or something

But with this annotation static analyser expect it to be not null, will give you, and any library user, a warning if you try to pass null to such parameter

Reply:
I mean usually to import things the dependency is required. Here we are importing from `org.jetbrains` - so asking how it works? I suppose it's not a part of Java distribution? Sorry, probably obvious thing for Java developer, but not for me :)

Reply:
ah, it comes from `org.jetbrains.annotations` package, and okhttp depends on it.

`com.google.code.findbugs:jsr305` has it's own version `javax.annotation.Nonnull` - they both work the same, but I guess it's more consistent using annotations from the same package, updated it

Reply:
Thanks, clear now!